### DISCUSSION TITLE
Best Practices for Coherently Combining Tokens in Language Models and Semantic Kernels

### DISCUSSION THREAD

**Discussion by jlopez362** [microsoft/semantic-kernel#5813] [Q&A] [answered]
Hello everyone, I'm new to the field of Language Models, AI, and Semantic Kernels. I understand that the generated words are separated, known as tokens. My question is: what is the best way to combine these words coherently? Within the connectors of the Semantic Kernel, is there anything that guides me on how to do this? I've been researching, but it's a bit complex to find a clear answer on this topic. Thank you for your assistance!

**Comment by sophialagerkranspandey** [comment, marked as answer]
Combining tokens into words is automatically handled by the model and is not something to worry about. Are there additional details that you can provide us with to proceed with answering?

**Reply by jlopez362** [reply]
Thank you very much for the response. I understand that if text is generated separately, it may be a limitation of the model rather than an issue with the Semantic Kernel
